pom.xml

```diff
@@ -118,6 +118,14 @@
         </dependency>
 
         <dependency>
+            <!-- For logging on Windows -->
+            <groupId>org.fusesource.jansi</groupId>
+            <artifactId>jansi</artifactId>
+            <version>1.13</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
No worries.  I will see if I can find some time to figure out what is going on.  Then, I'll clean things up.  :)